.github/stale.yml

```diff
@@ -1,63 +0,0 @@
-# Configuration for probot-stale - https://github.com/probot/stale
-
-# Number of days of inactivity before a stale Issue or Pull Request is closed.
-# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
-daysUntilClose: false
-
-# Number of days of inactivity before an Issue or Pull Request becomes stale
-daysUntilStale: 180 # 6 months
-
-# Issues with these labels will never be considered stale
-exemptLabels:
-  - "[Pri] High"
-  - "[Pri] BLOCKER"
-  - "[Type] Good For Community"
-  - "[Type] Good First Bug"
-  - "FixTheFlows"
-
-# Set to true to ignore issues in a project (defaults to false)
-exemptProjects: true
-
-# Set to true to ignore issues in a milestone (defaults to false)
-exemptMilestones: false
-
-# Label to use when marking an issue as stale
-staleLabel: "[Status] Stale"
-
-# Comment to post when closing a stale issue. Set to `false` to disable
-closeComment: false
-
-# Limit the number of actions per hour, from 1-30. Default is 30
-limitPerRun: 2
```

Comment:
This got dropped. If you want to add it back, looks like the corresponding input is `operations-per-run`.

Reply:
Ah yes, my bad. cf73d433761728929fe18497f39fae555b81dcf9